src/compas_rhino/conversions/surface.py

```diff
@@ -40,7 +41,30 @@ def geometry(self, geometry):
                 raise TypeError("Geometry cannot be interpreted as a `Rhino.Geometry.Surface`: {}".format(type(geometry)))
         self._geometry = geometry
 
-    def to_compas_mesh(self, cls=None, facefilter=None, cleanup=True):
+    def to_compas(self):
+        """Convert the surface to a COMPAS surface.
+
+        Returns
+        -------
+        :class:`compas_rhino.geometry.RhinoNurbsSurface`
+
+        Raises
+        ------
+        :class:`ConversionError`
+            If the surface BRep contains more than one face.
+        """
+        from compas.geometry import NurbsSurface
```

Comment:
any reason for this import to be inlined instead of at the top?

Reply:
was originally because of a circular import but perhaps this is not the case anymore. will investigate...